docs/pages/setup/security/reduce-blast-radius.mdx

```diff
@@ -0,0 +1,187 @@
+description: You can easily configure Teleport to minimize the scope of security breaches
+---
+
+Teleport encourages users to practice defense in depth so that every component of their infrastructure is protected against an attack, even if the attack is partially successful. You can configure Teleport to add layers of protection to your cluster when users authenticate or request elevated privileges. In this guide, we will show you how to:
+
+- [Make MFA mandatory for `tsh` logins](#make-mfa-mandatory-for-tsh-login)
+- [Present an MFA challenge for every attempt to access a resource](#present-an-mfa-challenge-for-every-attempt-to-access-a-resource)
+- [Require multiple reviews for role requests](#require-multiple-reviews-for-role-requests)
+- [Automatically prevent some roles from requesting others](#automatically-prevent-some-roles-from-requesting-others)
+
+## Make MFA mandatory for `tsh login`
+If a user sets up an account to authenticate to their Teleport cluster with only a password, an adversary can gain access to the password using brute-force attacks, person-in-the-middle attacks, or phishing. But even if a user's password is compromised, you can stop an attacker from authenticating with it when they run `tsh login`. 
+
+For clusters that use the local authentication connector, Teleport lets you make it mandatory for a user to enroll an MFA device when they create an account, and to authenticate using that device when they begin a new Teleport session. 
+
+To do so, make the following changes depending on your environment:
+
+<Tabs>
+<TabItem label="Self-hosted" scope={["oss","enterprise"]}>
+Add the following to your Teleport configuration file:
+
+```yaml
+auth_service:
+    authentication:
+        type: local
+        second_factor: otp|u2f|webauthn|on
+```
+</TabItem>
+<TabItem label="Teleport Cloud" scope={["cloud"]}>
+Create the following `cluster_auth_preference` dynamic resource:
+
+```yaml
+kind: cluster_auth_preference
+version: v2
+metadata:
+  name: cluster-auth-preference
+spec:
+  type: local
+  second_factor: otp|u2f|webauthn|on
+```
+
+Remember to create your dynamic resource using `tctl create -f <path to your YAML file>`.
+</TabItem>
+</Tabs> 
+
+To make MFA mandatory for all users, `second_factor` must be set to one of the following values:
+
+- `otp`
+- `u2f`
+- `webauthn`
+- `on`
+
+Choose `on` if you would like to require MFA for all users while letting them choose an OTP, U2F, or WebAuthn device. The other options restrict users to a single type of MFA device, which is useful for enforcing a particular standard of security. Once you start the Teleport Proxy Service with the `second_factor` configuration option set to one of these values, Teleport will mandate MFA by:
+
+- Adjusting the Teleport signup page so a user must enroll an MFA device of the kind you have selected. If the value of `second_factor` is `on`, users will have the option to select from multiple device types.
```

Comment:
Let's explain the process for enabling an MFA device for existing users when the cluster is updated from "MFA off" to "MFA on". I think we also have an `optional` setting which admins can use during the grace period.

Reply:
👍